### PR TITLE
Fixed `chia plotnft show` command

### PR DESCRIPTION
**Pull request by ChiaMineJP**
Fixed an issue where `chia plotnft show` command showed nothing if `-i/--id` option is not provided.
I believe that `chia plotnft show` changes introduced by this PR(#15704 ) is not fully expected so my PR is reverting behavior before the #15704 was merged.

### CODE REVIEW

**Review by arvidn — DISMISSED** [review]

File: chia/cmds/plotnft_funcs.py [hunk 1]
```diff
@@ -181,6 +181,29 @@ async def pprint_pool_wallet_state(
             print(f"Expected to leave after block height: {expected_leave_height}")
 
 
+async def pprint_all_pool_wallet_state(
```
Comment (jack60612) [inline chia/cmds/plotnft_funcs.py:184]: Extra p in print

Comment (jack60612) [reply chia/cmds/plotnft_funcs.py:184]: Wait isn't this function above?

**Review by jack60612 — CHANGES_REQUESTED** [review]
I think this merge / revert got messed up

File: chia/cmds/plotnft_funcs.py (hunk 1 above)
Comment (ChiaMineJP) [reply chia/cmds/plotnft_funcs.py:184]: `pprint` means pretty-print.
There is alerady `pprint_pool_wallet_state` function around line 122.
`pprint_all_pool_wallet_state` just runs `pprint_pool_wallet_state` through all pool wallet.

Comment (jack60612) [reply chia/cmds/plotnft_funcs.py:184]: pprint is still kinda stupid but sounds good.

**Review by jack60612 — DISMISSED** [review]
lgtm, last review sucked because i was on mobile.

**Review by jack60612 — APPROVED** [review]

**Review by jack60612 — APPROVED** [review]
lgtm, i cant see any new changes except the catch up with main

**Comment by ChiaMineJP** [comment]
@arvidn @jack60612 
Sorry I accidentally added commits which should belong to another PR.
I ran `git reset HEAD^` to dismiss the wrong commit and force-pushed the original commit-history which doesn't include the wrong commit.
The state of this PR is the same to the one both of you reviewed and approved.
Sorry for letting you review the same code twice!

**Comment by jack60612** [comment]
> @arvidn @jack60612 Sorry I accidentally added commits which should belong to another PR. I ran `git reset HEAD^` to dismiss the wrong commit and force-pushed the original commit-history which doesn't include the wrong commit. The state of this PR is the same to the one both of you reviewed and approved. Sorry for letting you review the same code twice!

all good, approved.

**Review by paninaro — APPROVED** [review]

**Pull request merged**